### DISCUSSION TITLE
Signtool bundle error.

### DISCUSSION THREAD

**Discussion by saiprabhur** [wixtoolset/issues#8752] [WiX Development]
Wix toolset version 3.11.2
We have bootstrapper application to create bundle setup.

<?xml version="1.0" encoding="utf-8"?>
<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension" xmlns:bal="http://schemas.microsoft.com/wix/BalExtension" xmlns:util="http://schemas.microsoft.com/wix/UtilExtension" xmlns:dep="http://schemas.microsoft.com/wix/DependencyExtension">
  <Bundle Name="MyApplicationSetup1.0.0.0" Copyright="MyApplicationSetup" DisableRemove="yes" DisableModify="yes" Manufacturer="MyApp" UpgradeCode="6f330b47-2577-43ad-9095-1861bb25899c" Version="1.0.0.0" dep:ProviderKey="01234567-8901-2345-6789-012345678901" >
    <BootstrapperApplicationRef Id="WixStandardBootstrapperApplication.HyperlinkLicense">
      <bal:WixStandardBootstrapperApplication SuppressOptionsUI="yes" ShowVersion="yes" ShowFilesInUse="yes" LicenseUrl="" />
    </BootstrapperApplicationRef>

    <bal:Condition Message="Setup could not be installed in 32 bit OS">VersionNT64</bal:Condition>

    <Chain>
      <MsiPackage DisplayInternalUI="no" Visible="yes" SourceFile="MyApplicationSetup.msi" InstallCondition="WINDOWS10_OR_NEWER_DETECTED" Compressed="yes" Vital="yes">
        <MsiProperty Name="WIXBUNDLEORIGINALSOURCE" Value="[WixBundleOriginalSource]" />
      </MsiPackage>
    </Chain>

    <util:RegistrySearchRef Id="RegistrySearch1" />
  </Bundle>

  <Fragment >
    <util:RegistrySearch Key="SOFTWARE\Microsoft\Windows NT\CurrentVersion" Value="CurrentMajorVersionNumber" Variable="WINDOWS10_OR_NEWER_DETECTED" Result="exists" Format="raw" Root="HKLM" Id="RegistrySearch1" />
  </Fragment>
</Wix>

The setup is created successfully. now we tried to digital signature the setup executable using signtool exe from command prompt.

"C:\Program Files (x86)\Windows Kits\10\bin\10.0.22621.0\x64\signtool.exe" sign /v /s MY /n "Digital Code Signing" /tr http://timestamp.digicert.com /td sha256 /fd sha256 /a "D:\Setup\MySetup.exe" 

The setup successfully digital signed.
When we run setup getting error below.

Error Screenshot
![image](https://github.com/user-attachments/assets/3b6496e8-26b1-4818-a41f-d6c00b5780e3)
Log files
[MyApplicationSetup1.0.0.0_20240916112851.log](https://github.com/user-attachments/files/17009073/MyApplicationSetup1.0.0.0_20240916112851.log)
[MyApplicationSetup1.0.0.0_20240916112906.log](https://github.com/user-attachments/files/17009075/MyApplicationSetup1.0.0.0_20240916112906.log)
[MyApplicationSetup1.0.0.0_20240916112910.elevated.log](https://github.com/user-attachments/files/17009076/MyApplicationSetup1.0.0.0_20240916112910.elevated.log)

Why this error thrown up for setup executable. Do we missing anything. how to resolve the issue.
When we do sign MSI its working without any issue.

**Comment by erikf-iccms** [comment]
I am not sure if this is related to your issue but it is not enough to just sign the setup executable/bundle. You also need to sign the Burn engine.
Look at the bottom of the page at this link to see how to sign a bundle correctly from either MSBuild or command line:
[https://wixtoolset.org/docs/tools/signing/#signing-bundles](https://wixtoolset.org/docs/tools/signing/#signing-bundles)
So try to sign both the engine and the bundle and see if it works.
